geode-core/src/main/java/org/apache/geode/internal/net/SSLConfigurationFactory.java

```diff
@@ -181,21 +181,18 @@ private SSLConfig createSSLConfig(final DistributionConfig distributionConfig,
     sslConfig.setRequireAuth(distributionConfig.getSSLRequireAuthentication());
     sslConfig.setAlias(distributionConfig.getSSLDefaultAlias());
     sslConfig.setUseDefaultSSLContext(distributionConfig.getSSLUseDefaultContext());
+
     return sslConfig;
   }
 
   private boolean determineIfSSLEnabledForSSLComponent(final DistributionConfig distributionConfig,
       final SecurableCommunicationChannel sslEnabledComponent) {
     if (ArrayUtils.contains(distributionConfig.getSecurableCommunicationChannels(),
-        SecurableCommunicationChannel.NONE)) {
-      return false;
-    }
-    if (ArrayUtils.contains(distributionConfig.getSecurableCommunicationChannels(),
-        SecurableCommunicationChannel.ALL)) {
+        SecurableCommunicationChannel.ALL))
```

Comment:
Good cleanup here.

However, per [the style guide](https://cwiki.apache.org/confluence/display/GEODE/Code+Style+Guide), "Always use braces, even around one-line `if`, `else` and other control statements."